lunabot_control/scripts/manual_controller.py

```diff
@@ -1,101 +1,187 @@
 #!/usr/bin/env python3
-import numpy as np
 import rospy
+
 from sensor_msgs.msg import Joy
+from lunabot_msgs.msg import RobotEffort
+
+from enum import Enum
+import numpy as np
 
 """
-.buttons
+.buttons   (1 is pressed, 0 not)
 0 A
 1 B
 2 X
 3 Y
 4 LB
 5 RB
-6 back
+6 back (view button)
```

Comment:
Wdym view button?

Reply:
![image](https://github.com/PurdueLunabotics/purdue_lunabotics/assets/112195326/9b9db5cf-d7db-45a5-b439-d8557d03e18c)

Reply:
Oh is that just the name of the button?